api/lib/infrastructure/serializers/jsonapi/session-serializer.js

```diff
@@ -64,6 +65,15 @@ module.exports = {
           }
         }
       },
+      assignedUser: {
+        ref: 'id',
+        ignoreRelationshipData: true,
+        relationshipLinks: {
+          related(record, current) {
+            return `/api/users/${current.id}`;
+          }
+        }
+      },
```

Comment:
<img width="419" alt="image" src="https://user-images.githubusercontent.com/38167520/78268139-11615100-7508-11ea-8ab5-d077e0e1b926.png">
 Direct / pas de relation ?

Reply:
Créer un repo pour les user pôle de certification
GET /api/member-of-pole-certif

Proposition de nom : 
- membre du pôle certification